docs/changelog.rst

```diff
@@ -1,7 +1,7 @@
 =========
 Changelog
 =========
-
+* :feature:`1925` Added the Power Index Pool Token (PIPT).
```

Comment:
For new assets we don't put them here but at the bottom of the changelog. I will make the change myself.